src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptionsSetup.cs

```diff
@@ -66,6 +71,21 @@ public void Configure(HttpWorkerOptions options)
         private void ConfigureWorkerDescription(HttpWorkerOptions options, IConfigurationSection workerSection)
         {
             workerSection.Bind(options);
+
+            var workerRuntime = _environment.GetFunctionsWorkerRuntime();
+            if (string.Equals(workerRuntime, RpcWorkerConstants.CustomHandlerLanguageWorkerName, StringComparison.OrdinalIgnoreCase))
+            {
+                options.CustomRoutesEnabled = true;
+            }
+
+            if (options.Http?.Routes is not null && options.Http.Routes.Any())
```

Comment:
I think `is not null` is redundant.

Reply:
Need the `is not null` the check if routes is not empty.

Reply:
You could do (I think)

```suggestion
            if (options.Http?.Routes is not null and not [])
```

or ...

```suggestion
            if (options.Http?.Routes?.Any() == true)
```

Reply:
I had this `options.Http?.Routes?.Any() == true` before. For some reason it did not work. Have left it as is for now. Will update it another time.